src/components/node-list/node-list.js

```diff
@@ -8,6 +9,7 @@ import './styles/node-list.css';
  * Scrollable list of toggleable items, with search & filter functionality
  */
 const NodeList = ({
+  isFaded,
```

Comment:
lol I had it as `faded` before and I changed it to this because I thought you'd think it was clearer this way 🙈 

idk whether we have an official standard for this but I thought a Boolean variable might be clearer with a verb?

Reply:
As far as I've seen it, variables are good for the prefix, but props and attributes tend to go without. I think a lot of that comes down from the style of HTML attributes, which do not include it.

Reply:
ok cool 🤷 

not too bothered, but I'll change it